package.json

```diff
@@ -16,7 +16,7 @@
     "lint:ci": "npm run lint -- --format junit --output-file /tmp/test-results/eslint/results.xml",
     "test": "TZ=Africa/Khartoum jest",
     "test:watch": "jest --watch",
-    "cypress:install": "npm install --no-save cypress@3.4.1 @percy/cypress@^0.2.3 atob@2.1.2",
+    "cypress:install": "npm install --no-save cypress@3.6.1 @percy/cypress@^2.2.0 atob@2.1.2",
```

Comment:
Maybe use a less strict version specifier for cypress?

Reply:
I usually don't pin it, but it seems to be often breaking something on those updates (already happen with a patch version as well). Perhaps just pin it with Patch version?

```suggestion
    "cypress:install": "npm install --no-save cypress@~3.6.1 @percy/cypress@^2.2.0 atob@2.1.2",
```

Reply:
Whatever you feel is best :-)